cfbs/git.py

```diff
@@ -18,6 +18,13 @@ class CFBSGitError(Exception):
     pass
 
 
+def check_git_exists():
+    try:
+        check_call(["git", "--version"], stdout=DEVNULL)
+    except CalledProcessError as cpe:
+        raise CFBSGitError("'git' was not found") from cpe
+
```

Comment:
Currently, it looks like this function will only raise a `CFBSGitError`. If it indeed can raise `FileNotFoundError`, I would expect at least a comment mentioning this.